modules/lib/src/main/resources/assets/styles/wizard/collaboration.less

```diff
@@ -4,7 +4,7 @@
 
 @space-between: 2px;
 
-@min-width: @unit-size + @border-width;
+@min-width: 26px; // @unit-size + @border-width;
```

Comment:
Isn't it still the same as `@unit-size + @border-width`? :)

Reply:
yes, but codacy was complaining :)